### PR TITLE
Fix/filesection requriments

### LINKED ISSUE #310: Explain when no fileSec's occur in section II, 5.3.5
The <fileSec> seems optional. Maybe it should be explained here in which specific cases an IP with no Data Objects could be transferred.

Section: II, 5.3.5
Text: 0..1 SHOULD

### LINKED ISSUE #321: More info regarding the folders
Why some folders are mandatory, although those can be empty (e.g. metadata folder)? Should there be some reason for this, it should be clarified.

### LINKED ISSUE #347: CSIP60
CSIP60: Should be clarified whether repository specific values can be used in addition to specified values (Documentation, Schemas and Representations)

Section: CSIP60

### LINKED ISSUE #403: Reformulation of CSIP66 and possible restructuring in fileSec requirements?
CS IP states:

CSIP66 | File `fileSec/fileGrp/file` | The lowest level file group (fileGrp) contains the file elements which describe the transferred file objects. When the file element is categorised as "Representations" each representation file group contains one file which is the reference to the METS document describing the representation  | 1..1 MUST
-- | -- | -- | --

I suggest we might reformulate to: 

CSIP66 | File` fileSec/fileGrp/file` | The lowest level file group (fileGrp) contains the file elements which describe the file objects in the file group.   | 1..n MUST
-- | -- | -- | --

3 changes in the reformulation:
1.  Change from _**the transferred file objects**_  to  _**file objects in the file group**_
Discussion: I don't think the word 'transferred file objects' is the best word to describe the file object in any given file group. 
2. Removal of _**When the file element is categorised as "Representations" each representation file group contains one file which is the reference to the METS document describing the representation**_
Discussion: I guess what we are trying to say here is something like: 
_"When there is representation data in the IP, then the root METS.xml SHOULD have a a file group where the USE-attribute has the value "Representations".  The file elements within this file group SHOULD describe and point to the METS.xml document in each representation and only to the representation METS.xml(s). In a representation METS.xml, then the the USE-attribute has the value "data" to describe the data files of the representation "_  I don't think that CSIP66 is the best place to describe this complex requirement. I think we might need a visualization and not a table description to solve this. I also the that we need to include "data" as a USE-attribute value in order to distinguish between representations and data files within representation. And also a better description of CSIP64 and the USE-attribute.   
3. Change from _**1..1 MUST**_ to  _**1..n MUST**_
Discussion: file elements can appear multiple times within a file group.

### LINKED ISSUE #404: CSIP60 and CSIP64
Regardles of the outcome of the [catalogue discussion](https://github.com/DILCISBoard/E-ARK-CSIP/issues/347) CSIP60 and CSIP64 seem redundant.

In CSIP60 it is not mentioned that it is via the  @USE attribute that the  naming (and classification) of the fileGroup takes place and vice versa - it is not mentioned in CSIP64 that this is related to CSIP60? If it is? Or is there something I have misunderstood?

### PR DESCRIPTION
**Pull request by karinbredenberg**
Update following these is that will be closed:
Closes #310 ,
Closes #321 ,
Closes #347 ,
Closes #403 ,
Closes #404

### CODE REVIEW

File: profile/CSIP.xml [hunk 1]
```diff
@@ -737,6 +737,7 @@
             <requirement ID="CSIP58" REQLEVEL="SHOULD" EXAMPLES="fileSecExample1 fileSecExample2">
                 <description>
                     <head>File section</head>
+                    <p xmlns="http://www.w3.org/1999/xhtml">The content being transferred is placed in the file section in different file groups (described in other requriments)</p>
```
Comment (carlwilson) [inline profile/CSIP.xml]: "requirments" is a typo.

File: profile/CSIP.xml [hunk 2]
```diff
@@ -758,17 +759,46 @@
...
-                      <p xmlns="http://www.w3.org/1999/xhtml">In one or more file groups with the catagorisation of "Documentation" all documentation pertaining to the transferred information is present.</p>
-                      <p xmlns="http://www.w3.org/1999/xhtml">In one or more file groups with the catagorisation of "Schemas" all XML-schemas pertaining to the transferred XML documents are present.</p>
-                      <p xmlns="http://www.w3.org/1999/xhtml">In one or more file groups with the catagorisation of "Representations" the data being transferred is present or in one file group the data for each representation is present.</p>
-                      <p xmlns="http://www.w3.org/1999/xhtml">To make the catagorisation easier the different files being transferred should be placed in folders with names following the catagorisation</p>
+                      <p xmlns="http://www.w3.org/1999/xhtml">There are one or more file group (fileGrp) elements present grouping the files in the main catagorisation of: Documentation, Schemas and Representations.</p>
```
Comment (carlwilson) [inline profile/CSIP.xml]: There are one or more file group (fileGrp) elements present, which divide the files into categories: Documentation, Schemas and Representations.

File: profile/CSIP.xml [hunk 3]
```diff
@@ -758,17 +759,46 @@
...
+                      <p xmlns="http://www.w3.org/1999/xhtml">These main file groups described are present even if there are no files referenced in the group to maintain the logical structure of the package.</p>
```
Comment (carlwilson) [inline profile/CSIP.xml]: The file groups MUST be present even if there are no referenced files to maintain the logical structure of the package.

File: profile/CSIP.xml [hunk 4]
```diff
@@ -758,17 +759,46 @@
...
+                      <p xmlns="http://www.w3.org/1999/xhtml">For each file group there are additional elements used which are described in their own requriments.</p>
```
Comment (carlwilson) [inline profile/CSIP.xml]: There are additional elements used for each file group, these are described in their own requirements

File: profile/CSIP.xml [hunk 5]
```diff
@@ -758,17 +759,46 @@
...
+                      <p xmlns="http://www.w3.org/1999/xhtml">Its possible to add other user defined file groups which then will use the same additional requriments.</p>
```
Comment (carlwilson) [inline profile/CSIP.xml]: It's possible for users to define additional file groups, these should comply with the same additional requirements.

File: profile/CSIP.xml [hunk 6]
```diff
@@ -758,17 +759,46 @@
...
                       <dl xmlns="http://www.w3.org/1999/xhtml">
                         <dt>METS XPath</dt><dd>fileSec/fileGrp</dd>
                         <dt>Cardinality</dt><dd>1..n</dd>
                     </dl>
                 </description>
             </requirement>
+            <requirement ID="CSIP113" REQLEVEL="MUST" RELATEDMAT="VocabularyFileGrpAndStructMapDivisionLabel CSIP64" EXAMPLES="fileSecExample1 fileSecExample2">
+                <description>
+                    <head>Documentations file grouping</head>
+                    <p xmlns="http://www.w3.org/1999/xhtml">In one or more file groups with the catagorisation of "Documentation" all documentation pertaining to the transferred content is present.</p>
```
Comment (carlwilson) [inline profile/CSIP.xml]: All documentation pertaining to the transferred content is present in one or more file groups categorised as "Documentation".

File: profile/CSIP.xml [hunk 7]
```diff
@@ -758,17 +759,46 @@
...
+                    <dl xmlns="http://www.w3.org/1999/xhtml">
+                        <dt>METS XPath</dt><dd>fileSec/fileGrp</dd>
+                        <dt>Cardinality</dt><dd>1..n</dd>
+                    </dl>
+                </description>
+            </requirement>
+            <requirement ID="CSIP114" REQLEVEL="MUST" RELATEDMAT="VocabularyFileGrpAndStructMapDivisionLabel CSIP64" EXAMPLES="fileSecExample1 fileSecExample2">
+                <description>
+                    <head>Schemas file grouping</head>
+                    <p xmlns="http://www.w3.org/1999/xhtml">In one or more file groups with the catagorisation of "Schemas" all XML-schemas pertaining to the transferred content in the form of XML documents are present.</p>
```
Comment (carlwilson) [inline profile/CSIP.xml]: All XML-schemas pertaining to the transferred content, in the form of XML documents, are present In one or more file groups with the categorisation "Schemas".

File: profile/CSIP.xml [hunk 8]
```diff
@@ -758,17 +759,46 @@
...
+                    <dl xmlns="http://www.w3.org/1999/xhtml">
+                        <dt>METS XPath</dt><dd>fileSec/fileGrp</dd>
+                        <dt>Cardinality</dt><dd>1..n</dd>
+                    </dl>
+                </description>
+            </requirement>
+            <requirement ID="CSIP115" REQLEVEL="MUST" RELATEDMAT="VocabularyFileGrpAndStructMapDivisionLabel CSIP64" EXAMPLES="fileSecExample1 fileSecExample2">
+                <description>
+                    <head>Representations file grouping</head>
+                    <p xmlns="http://www.w3.org/1999/xhtml">In one or more file groups with the catagorisation of "Representations" the pointer to the METS document describing the representation or the content (data) being transferred is present.</p>
```
Comment (carlwilson) [inline profile/CSIP.xml]: A pointer to the METS document describing the representation or the content being transferred must be present in one or more file groups with the categorisation "Representations" .

File: profile/CSIP.xml [hunk 9]
```diff
@@ -822,11 +852,11 @@
             <requirement ID="CSIP66" REQLEVEL="MUST" EXAMPLES="fileSecExample1 fileSecExample2">
                 <description>
                     <head>File</head>
-                    <p xmlns="http://www.w3.org/1999/xhtml">The lowest level file group (fileGrp) contains the file elements which describe the transferred file objects.</p>
-                    <p xmlns="http://www.w3.org/1999/xhtml">When the file element is categorised as "Representations" each representation file group contains one file which is the reference to the METS document describing the representation.</p>
+                    <p xmlns="http://www.w3.org/1999/xhtml">The file group (fileGrp) contains the file elements which describe the file objects.</p>
+                    <p xmlns="http://www.w3.org/1999/xhtml">When the METS document descibes as "Representations" each representation file group contains the reference to the METS document describing the representation.</p>
```
Comment (carlwilson) [inline profile/CSIP.xml]: I'm not sure what 856 is driving at

File: specification/implementation/metadata/mets/filesec/index.md [hunk 10]
```diff
@@ -3,6 +3,6 @@ Use of the METS fileSec element is highly recommended by the CSIP (although not
...
+The main requirement of the CSIP is that the file section of both the root and representation METS files includes at least one file group (element fileGrp) grouping together files.  CSIP structures the different files into structural components (i.e. documentation, schemas, data) which are described by its own fileGrp element. Representations including their own METS files, the components (including data files) of a representation should be described only in the representation METS. The root METS file should still include a fileGrp for each representation but only reference the METS.xml file of the representation to ensure knowledge about the location and checksum.
```
Comment (carlwilson) [inline specification/implementation/metadata/mets/filesec/index.md]: ensure that the location and checksum are known

File: profile/CSIP.xml [hunk 11]
```diff
@@ -759,10 +759,10 @@
...
                   <description>
                     <head>File grouping</head>
-                      <p xmlns="http://www.w3.org/1999/xhtml">There are one or more file group (fileGrp) elements present grouping the files in the main catagorisation of: Documentation, Schemas and Representations.</p>
-                      <p xmlns="http://www.w3.org/1999/xhtml">These main file groups described are present even if there are no files referenced in the group to maintain the logical structure of the package.</p>
-                      <p xmlns="http://www.w3.org/1999/xhtml">For each file group there are additional elements used which are described in their own requriments.</p>
-                      <p xmlns="http://www.w3.org/1999/xhtml">Its possible to add other user defined file groups which then will use the same additional requriments.</p>
+                      <p xmlns="http://www.w3.org/1999/xhtml">There are one or more file group (fileGrp) elements present, which divide the files into categories: Documentation, Schemas and Representations.</p>
```
Comment (koit) [inline profile/CSIP.xml]: There is a conflict here:
>There are one or more file group (fileGrp) elements present

vs line 763:
>The file groups MUST be present even if there are no referenced files

If some file groups are mandatory, there should be an exhaustive list of them, e.g.:

"The fileSec MUST have one or more file group (fileGrp) elements, which divide the files into categories. To maintain the logical structure of the package, file groups for the categories "Documentation," "Schemas" and "Representations" MUST be present even if there are no files referenced in them."

Comment (koit) [reply profile/CSIP.xml]: [VocabularyFileGrpAndStructMapDivisionLabel](http://earkcsip.dilcis.eu/#VocabularyFileGrpAndStructMapDivisionLabel) is about these categories - maybe the whole vocabulary should have mandatory file groups? Currently, there are the three above mentioned values and "Metadata."

Comment (carlwilson) [reply profile/CSIP.xml]: No proposing getting rid of CSIP60 altogether as it's kind of redundant.

File: profile/CSIP.xml [hunk 12]
```diff
@@ -755,15 +756,31 @@
...
+                    <p xmlns="http://www.w3.org/1999/xhtml">All documentation pertaining to the transferred content is placed in one or more file group elements with @USE attribute value "Documentation".</p>
+                    <dl xmlns="http://www.w3.org/1999/xhtml">
+                        <dt>METS XPath</dt><dd>fileSec/fileGrp</dd>
+                        <dt>Cardinality</dt><dd>1..n</dd>
+                    </dl>
+                </description>
+            </requirement>
+            <requirement ID="CSIP113" REQLEVEL="MUST" RELATEDMAT="VocabularyFileGrpAndStructMapDivisionLabel CSIP64" EXAMPLES="fileSecExample1 fileSecExample2">
+                <description>
+                    <head>Schema file group</head>
```
Comment (koit) [inline profile/CSIP.xml]:
```suggestion
                    <head>Schemas file group</head>
```

File: profile/CSIP.xml (hunk 11 above)
Comment (koit) [reply profile/CSIP.xml]: I feel I lack a strong enough grasp of the whole picture to take position on this. After splitting `fileSec/fileGrp` into "Documentation," "Schemas" and "Representations," a generic `fileSec/fileGrp` requirement seems unnecessary, but there might still be some useful general things about `fileSec/fileGrp` that are best stated in a separate requirement. E.g. regarding structuring: nesting file groups to mirror the folder structure vs presenting them in a flat list. So I'd add it to the todo list of the dedicated task force I suggested in #426.

**Pull request merged**